cmd/exporter/main.go

```diff
@@ -8,28 +8,35 @@ import (
 	"github.com/kuskoman/logstash-exporter/config"
 	"github.com/kuskoman/logstash-exporter/server"
 	"github.com/prometheus/client_golang/prometheus"
+	"golang.org/x/exp/slog"
 )
 
 func main() {
-	warn := godotenv.Load()
-	if warn != nil {
-		log.Println(warn)
+	err := godotenv.Load()
```

Comment:
since this error is not a critical error, leading to panic i would like you to leave name of the variable as "warn"

Reply:
I do not agree on this, since the returned interface is an `error` I would like to use the variable name `err,` even if it's a trivial error. The problem lies at first glance, I didn't think this was an `error` type. 

But in the end this is just my opinion, will of course change it to warn again :)

Reply:
You can change it to something like erorr and criticalError
But I would like to distingiush these two, as godotenv load can safely fail and logger should crash the application in my opinion